src/machines/java/com/enderio/machines/common/blockentity/EnchanterBlockEntity.java

```diff
@@ -26,7 +26,7 @@
 public class EnchanterBlockEntity extends MachineBlockEntity {
 
     private final RecipeWrapper container;
-
+    private Optional<EnchanterRecipe> currentRecipe;
```

Comment:
make field nullable

Reply:
instead of optional? The reason I'm using it is cause this comes directly from the recipe manager. Though I can make it nullable instead.